test/client.ts

```diff
@@ -0,0 +1,71 @@
+import { testServer } from './test-server';
+import { OAuth2Client } from '../src';
+import { expect } from 'chai';
+
+describe('OAuth2Client', () => {
+  const server = testServer();
+
+  describe('Token revocation', () => {
+    it('should revoke access token when requested', async () => {
```

Comment:
Is this `it` intentional here? Looks like there isn't a specific `expect` for this function, rather it has nested describes?

Reply:
Should probably be a describe